packages/yswaps/scripts/libraries/types.ts

```diff
@@ -2,17 +2,21 @@ import { BigNumber, PopulatedTransaction } from 'ethers';
 import { TradeFactory } from '@typechained';
 import { MainnetSolvers } from '../configs/mainnet';
 import { FantomSolvers } from '@scripts/configs/fantom';
+import { Network as EthersNetwork } from '@ethersproject/networks';
 
 export type DexLibrarySwapProps = {
   tokenIn: string;
   tokenOut: string;
   amountIn: BigNumber;
+  strategy: string;
   hops?: string[];
 };
 
 export type DexLibrarySwapResponse = {
-  data: string;
+  dex: string;
+  executionTransactionData: string;
```

Comment:
is the dex in charge of generating the execution tx data? Should this be the solver after comparing all the dexes outputs?